<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>交易管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="currentUserLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前用户：未登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="balanceLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>781</width>
      <height>461</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>商品名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>种类</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>商品描述</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>售价</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>余量</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>商家</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="filterLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>517</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>筛选</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountLine">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>520</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>520</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>输入数量：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCartButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>517</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>加入购物车</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="zhanghu">
    <property name="title">
     <string>账户</string>
    </property>
    <addaction name="dengluzhuce"/>
    <addaction name="xiugaimima"/>
    <addaction name="actionrecharge"/>
    <addaction name="actioncart"/>
    <addaction name="actiontuichu"/>
   </widget>
   <widget class="QMenu" name="guanli">
    <property name="title">
     <string>管理</string>
    </property>
    <addaction name="actionaddGoods"/>
    <addaction name="actionmanage"/>
   </widget>
   <addaction name="zhanghu"/>
   <addaction name="guanli"/>
  </widget>
  <action name="dengluzhuce">
   <property name="text">
    <string>登录或注册</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="xiugaimima">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actiontuichu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>退出登录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionaddGoods">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>添加商品</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionmanage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>管理商品</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionrecharge">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>充值</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actioncart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>购物车</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
